src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -89,6 +95,13 @@ public TenantCollection getAll(UUID consortiumId) {
     return result;
   }
 
+  @Override
+  public TenantDetails getTenantDetailsById(UUID consortiumId, String tenantId) {
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+    checkTenantExistsOrThrow(tenantId);
+    return converter.convert(tenantDetailsRepository.findById(tenantId).orElse(null), TenantDetails.class);
```

Comment:
Converter can throw NPE here is tenant will not be found

Reply:
thanks, updated